STYLEGUIDE.md

```diff
@@ -168,11 +168,13 @@ window.clearTimeout(1);
 document.createElement('div');
 ```
 
-### Use `useIsomorphicLayoutEffect` instead of `useLayoutEffect`
+### Import `useLayoutEffect` from `utils` instead of using `React.useLayoutEffect`
```

Comment:
Added explanation.

Reply:
nit: Maybe something shorter than my previous suggestion would be better?

```diff
- ### Import `useLayoutEffect` from `utils` instead of using `React.useLayoutEffect`, since `useLayoutEffect` from `utils` is SSR safe
+ ### Import `useLayoutEffect` (SSR-safe) from `utils` instead of using `React.useLayoutEffect` (client only)
```

Reply:
Changed in latest commit.